cmd/ctr/commands/run/run_unix.go

```diff
@@ -329,6 +330,19 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 			})
 		}
 
+		if c := context.String("blockio-config-file"); c != "" {
```

Comment:
@askervin 
Re https://github.com/kubernetes/kubernetes/issues/92287#issuecomment-1225239183
I see this is where it should re-read the blockio configuration file and update the list of devices, but it doesn't look like `"blockio-config-file"` is set unless creating a container with the `ctr` cli?

Reply:
@d-honeybadger, thank you for digging into this! I created PR #7418 to fix this issue.